src/ic-cdk/src/api/management_canister/http_request.rs

```diff
@@ -131,6 +131,8 @@ pub struct CanisterHttpRequestArgument {
     /// The requested URL.
     pub url: String,
     /// The maximal size of the response in bytes. If None, 2MiB will be the limit.
+    /// This value affects the cost of the http request and it is highly recommended
+    /// to set it as low as possible to avoid unnecessary extra costs.
```

Comment:
Do you think it makes sense to add a link to the pricing section of the [documentation](https://internetcomputer.org/docs/current/developer-docs/integrations/http_requests/http_requests-how-it-works#pricing)?

Reply:
Well, it wouldn't hurt, so I'll add it.